docs/_includes/section-notice.html

```diff
@@ -98,6 +98,18 @@ <h3 id="section-notice-status">Status Section Notice</h3>
     {% endhighlight %}
 
     <h3 id="section-notice-educational">Educational Section Notice</h3>
+
+    <section class="section-notice" role="region" aria-label="Attention" aria-roledescription="Notice">
+        <div class="section-notice__header" id="section-notice-attention">
+            <svg class="icon icon--attention-filled-16" height="16" width="16" aria-label="Attention">
+                {% include symbol.html name="attention-filled-16" %}
+            </svg>
+        </div>
+        <div class="section-notice__main">
+            <p>This variation of section notice is deprecated in lieu of a separate education notice module. Please use/migrate to that module instead as this variation will be removed in the next major release.</p>
+        </div>
+    </section>
+
```

Comment:
Lets remove all the old of way of doing this and add a message if you want to use education notice variant to use the component.

Reply:
I don't mind doing that, but would that be a breaking change for those using it the old way currently? We're ok with that?

Reply:
We're simply removing the code from the docs so people don't use it
The less code should still be around until next major. Maybe add a deprecated comment next to the code in less
It wont be breaking